cart.py

```diff
@@ -1,6 +1,94 @@
+"""
+Classes
+-------
+Product
+Cart(Product)
+"""
+import csv
+
+
 class Product:
-    pass
+    """
+    Attributes
+    ----------
+    file_name : str
+        name of the file which contains products
+    volume : str
+        all products from the file_name file
+    num_of_prod : int
+        number of the products
+
+    Methods
+    -------
+    get_product(file_name)
+    name() -- Return name of the product
+    price() -- Return price of the product
+    qty() -- Return quantity of the products
+    """
+
+    def __init__(self, file_name):
+        self.file_name = file_name
```

Comment:
working with a file is not the responsibility of the Product class. Try to move it to the Cart.